rotkehlchen/assets/exchanges_mappings/binance.py

```diff
@@ -144,4 +144,6 @@
     evm_address_to_identifier('0x320623b8E4fF03373931769A31Fc52A4E78B5d70', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'RSR',  # noqa: E501
     evm_address_to_identifier('0xa2E3356610840701BDf5611a53974510Ae27E2e1', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'WBETH',  # noqa: E501
     evm_address_to_identifier('0xfFffFffF2ba8F66D4e51811C5190992176930278', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'COMBO',  # noqa: E501
+    evm_address_to_identifier('0x72aDadb447784dd7AB1F472467750fC485e4cb2d', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'WLD',  # noqa: E501
```

Comment:
Is this really WLD? The address seems to point to https://etherscan.io/address/0x72aDadb447784dd7AB1F472467750fC485e4cb2d which is Worldcore, not worldcoin. And when I try to go to the site, it points to a scam that brave warns me about